test/integration/driver.go

```diff
@@ -48,8 +48,8 @@ const (
 	// budget is the maximum number of retries with 1s delays
 	budget = 90
 
-	// Mixer SHA *update manually*
-	mixerSHA = "ea3a8d3e2feb9f06256f92cda5194cc1ea6b599e"
+	// Mixer tag *update manually*
+	mixerTag = "demo"
```

Comment:
I am not a fan of dynamic tags - can you put SHA instead. I don't want to be debugging mixer problems when I make changes that don't touch it.